// ss protocol gateway to gateway_mgr

syntax = "proto3";

enum SSGWGMID
{
	SSID_GW_GM_INIT = 0;
	SSID_GW_GM_HEAT_BEAT_REQ = 1;
	SSID_GM_GW_HEAT_BEAT_RSP = 2;
}

message SSGWGMInit
{
}

message SSGWGMHeartBeatReq
{
	uint32 peers_num = 1;
}

message SSGMGWHeartBeatRsp
{

}

message SSGWGMPkgBody
{
	oneof body
	{
		SSGWGMInit init = 1;
		SSGWGMHeartBeatReq heart_beat_req = 2;
		SSGMGWHeartBeatRsp heart_beat_rsp = 3;
	}
}